ctapipe/image/hillas.py

```diff
@@ -186,27 +143,20 @@ def hillas_parameters_1(pix_x, pix_y, image):
     # pix_x[np.argmax(image)]) * cos_delta + (mean_y -
     # pix_y[np.argmax(image)]) * sin_delta
 
-    # Compute azwidth by transforming to (p, q) coordinates
-    sin_theta = mean_y / r
-    cos_theta = mean_x / r
-    q = (mean_x - pix_x) * sin_theta + (pix_y - mean_y) * cos_theta
-    m_q = np.sum(image * q) / size
-    m_qq = np.sum(image * q * q) / size
-    azwidth_2 = m_qq - m_q * m_q
-    azwidth = np.sqrt(azwidth_2)
-
-    return (MomentParameters(size=size,
-                             cen_x=mean_x,
-                             cen_y=mean_y,
-                             length=length,
-                             width=width,
-                             r=r,
-                             phi=phi,
-                             psi=delta,
-                             miss=miss),
-            HighOrderMomentParameters(Skewness=skewness,
-                                      Kurtosis=kurtosis,
-                                      Asymmetry=asym))
+    HillasContainer = Container("HillasParams")
+    HillasContainer.add_item("size", size)
+    HillasContainer.add_item("cen_x", mean_x * unit)
+    HillasContainer.add_item("cen_y", mean_y * unit)
+    HillasContainer.add_item("length", length * unit)
+    HillasContainer.add_item("width", width * unit)
+    HillasContainer.add_item("dist", r * unit)
+    HillasContainer.add_item("phi", (phi * u.rad).to(u.deg))
+    HillasContainer.add_item("psi", (psi * u.rad).to(u.deg))
+    HillasContainer.add_item("miss", miss * unit)
+    HillasContainer.add_item("Skewness", skewness)
+    HillasContainer.add_item("Kurtosis", kurtosis)
+    HillasContainer.add_item("Asymmetry", asym)
+    return HillasContainer
```

Comment:
This is not how containers work - a HillasContainer class should be defined in io/containers.py  (also you will need to pull in the latest changes, since the Container definition no longer uses `add_item`. Otherwise it looks fine.